addons/website_slides_survey/controllers/slides.py

```diff
@@ -153,7 +153,7 @@ def _prepare_ranks_badges_values(self, **kwargs):
 
         # 3. Remove certification badge from badges and keep only certification badge linked to opened survey
         badges = values['badges'] - certification_badges
-        certification_badges = certification_badges.filtered(lambda b: b.survey_id.state == 'open')
+        certification_badges = certification_badges.filtered(lambda b: b.survey_id.state is True)
```

Comment:
There is some confusion between your commit message and the code I see here.

You say "state" was replaced by a check on the "active" field, yet you still check "state" ?